README.md

```diff
@@ -23,6 +23,17 @@ CMD: `racket`
 These images, tagged with `-full` at the end, use the full Racket
 distribution.
 
+#### "CS" images
```

Comment:
Can you rename this section to "Racket-on-ChezScheme images"? Makes it easier to tell what these are when just skimming the readme.